script/GameObjects/Components/Economy/Economy.lua

```diff
@@ -62,27 +76,34 @@ function Economy:update (dt)
 --      end
 --    end
 
-    -- Cache profitable trade offers
+    -- Cache profitable trade jobs
+    local alljobcount = 0
+    local realjobcount = 0
```

Comment:
same here

Reply:
Change made, update in follow-up commit.